plugins/CoreAdminHome/stylesheets/trackingCodeGenerator.less

```diff
@@ -0,0 +1,44 @@
+.trackingCodeAdvanceOptions {
+  .advance-option {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 21px;
+    margin-top: 2rem;
+    margin-bottom: 1.5rem;
+
+    a {
+      text-decoration: none !important;
+      color: @color-green-matomo;
+
+      &:focus {
+        outline: unset !important;
+      }
+    }
+  }
+
+  .hr-bottom {
+    margin-top: 1rem;
+    margin-bottom: 2rem;
+  }
+
+  .chevron {
```

Comment:
Since we might want to use the chevron up / down style elsewhere in the future, would it perhaps be worth adding these glyphs to the Morpheus icon font rather than using inline SVGs?

Reply:
Yep, we should do that. Also the up and down should just be an applied transformation (rotate 180 degrees), not a separate image.

Reply:
Added glphs, thanks @bx80 for the help in adding the icons